package.json

```diff
@@ -11,6 +11,7 @@
     "displayName": "ROS",
     "description": "Develop Robot Operating System (ROS) with Visual Studio Code.",
     "icon": "media/icon.png",
+    "aiKey": "6a0f99ed-47cf-4ce0-b2e5-321ef7a6aa67",
```

Comment:
Put this in the pipeline instead of in github.

Reply:
Use a different key for pipeline builds from github builds.

Reply:
created https://github.com/ms-iot/vscode-ros/issues/60 to track this